### DISCUSSION TITLE
Exporting as .png?

### DISCUSSION THREAD

**Discussion by nshiab** [observablehq/plot#1933] [Q&A]
On Observable notebooks, we can export charts as .png. How do you do that? I am just curious. 🤓 

I am working on a JavaScript library that uses Observable Plot, and I would like to offer the same option.

Here's what I have for now: https://github.com/nshiab/simple-data-analysis/blob/254-duckdb-wasm/src/functions/writeChart.ts

Thanks for your amazing work! :)

**Comment by Fil** [comment]
https://github.com/bubkoo/html-to-image works quite well; see https://observablehq.com/@fil/hello-html-to-image.

**Reply by nshiab** [reply]
Thanks @Fil! J'espère que tu vas bien. Toujours aussi fan de ton boulot! :)
